README.md

```diff
@@ -12,3 +12,23 @@ Check out the [Issues](https://github.com/webcomponents/community-protocols/issu
 | Proposal        | Author         | Status |
 |-----------------|----------------|--------|
 | TK | TK | TK     |
+
+## Status
+
+Community Protocols will go through a number of phases during which they can gather further community insight and approval before becoming "Accepted". These phases are as follows:
+
+- *Proposal*
+  "Proposal" status applies to just about anything that community members are interested in putting some thought into. While an issue submitted to this repo can help generate initial ideas on the protocol or space of interest and clarify the information needed to kick off a more fruitful conversation, a PR will serve to make known that you are interested in support to drive the protocol in question forward. Having an initial explainer included in this PR, while adding the protocol to the "Proposals" table shown above, will prepare the community to both communicate about and contribute to the development of the protocol asynchronously.
+
+- *Candidate*
```

Comment:
I thought we had said that candidate was after draft and the last call for feedback before becoming accepted, somewhat like TC39: https://tc39.es/process-document/

Reply:
I may have my notes wrong here. I actually have two swap marks on that as I remember going back and forth on it last minute. Does anyone else have a preference? Happy to swap it if that's the correct path!

Reply:
I think aligning with TC39 will be easiest, since it's pretty well-known.

Reply:
Swapped.